CONTRIBUTING.rst

```diff
@@ -13,42 +13,28 @@ TL;DR
 
 .. code-block:: console
 
-    $ # Export libpq env vars for PG connection
-    $ export PGDATABASE=procrastinate PGHOST=localhost PGUSER=postgres
-
-    $ # Launch PostgreSQL within Docker
+    $ # Launch services within Docker
     $ docker-compose up -d
```

Comment:
When doing `docker-compose up -d` the first time the procrastinate worker will fail to start because the Procrastinate database objects have not been created yet. No?

Reply:
As you suggested on meet, I separated the commands to first launch only postgres and run the migrations, before starting the worker.